Aztec/Classes/TextKit/ColorProvider.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+import UIKit
+
+/// A class that provides colors to the formatter in TextKit.
+/// By default only the standard html colors are provided, but new color can be added by using the `add(namedColors:)` method.
+public class ColorProvider {
+
+
+    private init() {
+
+    }
+
+    public static var shared = ColorProvider()
+
+    private lazy var htmlColors: [String: UIColor] = {
+        guard let colorDataURL = Bundle.aztecBundle.url(forResource: "html_colors", withExtension: "json"),
+            let data = try? Data(contentsOf: colorDataURL),
+            let object = try? JSONSerialization.jsonObject(with: data, options: .allowFragments),
+            let colorDict = object as? [String: String] else {
+                return [String:UIColor]()
+        }
+
+        return colorDict.compactMapValues { (value) -> UIColor? in
+            return UIColor.init(hexString: value)
+        }
```

Comment:
This could be written as:
`return colorDict.compactMapValues { UIColor(hexString: $0) }`
A bit more direct and still readable.
If you prefer to keep it as it is (which is also fine), the `init` is still not needed 👍